x-pack/plugins/observability/public/typings/slo/index.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+interface SLO {
```

Comment:
💬 need to find a way to map this from the response schema defined on the server.

Reply:
I am wondering when we use the `types.ts` file defined in the page when we sue the `typings` directory in public, any idea?
@CoenWarmer Maybe we can include this part in the plugin structure as well.

Reply:
Ah you mean using the global common `typings` vs the specific page `types`. I understand. I guess for now this should be moved inside the page's types 👍🏻 
Later, it might move outside since we gonna have other SLO related pages which will reuse the SLO type.